lib/control.ml

```diff
@@ -1,4 +1,6 @@
-(* Funtor, Applicative and Monad module types *)
+open Any
+
+(* Functor, Applicative and Monad module types *)
 module type Functor = sig
   type 'a t
```

Comment:
```suggestion
  type +'a t
```

(and similarly throughout, since `Monad`, `Functor`, `Applicative`, etc. are all covariant)